CHANGES_3.0.0.md

```diff
@@ -177,7 +173,14 @@ in on the options object . Additionally, `find` does not support individual opti
 `limit` as positional parameters. You must either pass in these parameters in the `options` object,
 or add them via `Cursor` methods like `Cursor.prototype.skip`.
 
-### Aggregation
+### `aggregate`
```

Comment:
`Collection.prototype.aggregate` ? Or just `Aggregation`? I'm not too picky though